contracts/SignatureReg.sol

```diff
@@ -16,43 +16,42 @@
 
 pragma solidity ^0.4.1;
 
-import "./Owned.sol";
 
-
-contract SignatureReg is Owned {
+contract SignatureReg {
 	// mapping of signatures to entries
 	mapping (bytes4 => string) public entries;
 
 	// the total count of registered signatures
 	uint public totalSignatures = 0;
 
-	// allow only new calls to go in
-	modifier whenUnregistered(bytes4 _signature) {
-		if (bytes(entries[_signature]).length != 0)
-			return;
-		_;
-	}
-
 	// dispatched when a new signature is registered
 	event Registered(address indexed creator, bytes4 indexed signature, string method);
 
 	// constructor with self-registration
-	constructor() public {
+	constructor()
+		public
+	{
 		register("register(string)");
 	}
 
 	// registers a method mapping
-	function register(string _method) public returns (bool) {
+	function register(string _method)
+		public
+		returns (bool)
+	{
 		return _register(bytes4(keccak256(_method)), _method);
 	}
 
-	// in the case of any extra funds
-	function drain() onlyOwner public {
```

Comment:
Are you sure we would never need this?
Seems like a useful function, I can imagine us getting extra ETH on such a contract every now and then.

Reply:
The reason I removed this is because there's no way to get ETH into this contract since no function is declared as `payable`.

Reply:
Makes sense. LGTM then